hazelcast/src/test/java/com/hazelcast/cluster/ClusterJoinConfigCheckTest.java

```diff
@@ -52,28 +51,6 @@ public void killAllHazelcastInstances() throws IOException {
     }
 
     @Test
-    public void tcp_whenGroupPasswordMismatch_thenNewNodeIsShutDown() {
```

Comment:
We can add some tests verifying that the password is empty and some tests for verifying that some custom byte array processor was called.